test/passing/tag_only.ml

```diff
@@ -126,11 +126,11 @@ module A : sig
 end =
   B
 
-open Module.With_veryyyyyy_loooooooooooooooooooooooong_naaaaaaaaaaaaaaaaame
 (** @deprecated  *)
+open Module.With_veryyyyyy_loooooooooooooooooooooooong_naaaaaaaaaaaaaaaaame
 
-include Module.With_very_loooooooooooooooooooooooong_naaaaaaaaaaaaaaaaame
 (** @deprecated  *)
+include Module.With_very_loooooooooooooooooooooooong_naaaaaaaaaaaaaaaaame
 
-module A = Module.With_very_loooooooooooooooooooooooong_naaaaaaaaaaaaaaaaame
 (** @deprecated  *)
+module A = Module.With_very_loooooooooooooooooooooooong_naaaaaaaaaaaaaaaaame
```

Comment:
Since the module name is long I think it makes sense to put the docstring before.

Reply:
I disagree. It feels arbitrary and may be reported as a bug in the future.

Reply:
Fixed.